src/main/MQ2ImGuiConsole.cpp

```diff
@@ -1328,6 +1348,49 @@ static void MakeColorGradient(float frequency1, float frequency2, float frequenc
 }
 
 //----------------------------------------------------------------------------
+ImGuiWindowFlags ConsoleFlagSetup()
+{
+	ImGuiWindowFlags windowFlags = 0;
+	if (!s_noTitleBar || s_mouseHoveringConsoleTimer != 0) windowFlags |= ImGuiWindowFlags_MenuBar;
+	if (s_noTitleBar)  windowFlags |= ImGuiWindowFlags_NoTitleBar;
+	if (s_lockConsole) windowFlags |= ImGuiWindowFlags_NoMove | ImGuiWindowFlags_NoResize;
+	return windowFlags;
+}
+
+void AlphaSetting(bool AlphaBool)
```

Comment:
Similar feedback on function and variable names, it needs some revising.

However, the whole function can be elimininated and just use a one liner at the call site:

```
ImGui::SetNextWindowBgAlpha(m_alphaBool || !s_alphaBool || s_mouseHoveringConsoleTimer != 0 ? 1.0f : .1f);
```